bin/makeDocs

```diff
@@ -156,18 +162,18 @@ def main(topProductName, products, htmlDir, useDot=True):
                 configs[p].entries = dict([(k, v) for k, v in configs[p].entries.items()
                                            if k in ("EXCLUDE", "INPUT", "PROJECT_NAME",)])
 
-        except RuntimeError, e:
+        except RuntimeError as e:
             if p == "lsstDoxygen":
-                print >> sys.stderr, e
+                print(e, file=sys.stderr)
                 sys.exit(1)
-            
-            print >> sys.stderr, e
 
-    config = Doxyfile(" ".join(productList), configFile=None) # Just init the object, no config file
+            print(e, file=sys.stderr)
```

Comment:
Yes. I might make that tweak even though it's arguably outside the remit of the ticket.